### PR TITLE
Update Website README.md

### LINKED ISSUE #6: Document the process of adding a dependency
### Purpose

To help users with the process of adding a new dependency to the website. 

### Changes

We'll create a new header in the README with the title **Adding Dependencies**.  Whenever a new dependency is added to the `package.json` file, the user should make sure to run `npm install` and commit the autogenerated file `package-lock.json`. 

### Resources

- [Commiting package-lock.json is a standard practice](https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5)

### PR DESCRIPTION
**Pull request by jahn18**
### Description
Updates the README for the website 

- Resolves #6 

### Verification

- [ ] 

### Resources

-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,98 +4,32 @@ In the website development timeline, we are currently evaluating the folllowing
...
+developers and for the cloud era. If you want to learn more about MongoDB, visit their docs site: [MongoDB Documentation](https://docs.mongodb.com/).
 
-If you want to learn more about MongoDB, visit the following pages:
+## Set Up
 
-- [MongoDB Atlas](https://mongodb.com/atlas)
-- [MongoDB Documentation](https://docs.mongodb.com/)
+1. Set each variable in `.env.local`:
+    - `MONGODB_URI` - Your MongoDB connection string: `mongodb://<ip-address-of-workspace-container>:27017/testdb`
+        - To find the ip address of the workspace container:
```
Comment (patrick-5546) [inline README.md:17]: Would it be better to use localhost and forward the port in the sailbot workspace docker files?

**Review by patrick-5546 — COMMENTED** [review]
One comment about lint errors, if you make your changes in the sailbot workspace dev container, you will be able to see and fix the errors before you commit and push

**Comment by jahn18** [comment]
> One comment about lint errors: if you make your changes in the sailbot workspace dev container, you will be able to see and fix the errors before you commit and push

Got it, thanks for the feedback

File: README.md (hunk 1 above)
Comment (jahn18) [reply README.md:17]: O yes - that would make things a lot easier. Let me try that

**Comment by patrick-5546** [comment]
> > One comment about lint errors: if you make your changes in the sailbot workspace dev container, you will be able to see and fix the errors before you commit and push
> 
> Got it, thanks for the feedback and for fixing the lint errors 😄

No problem! One more thing, if you still want to edit in the browser you can do that through github codespaces: https://ubcsailbot.github.io/docs/main/current/sailbot_workspace/setup/#setup-sailbot-workspace-in-a-github-codespace

File: README.md (hunk 1 above)
Comment (jahn18) [reply README.md:17]: PR to port forward: https://github.com/UBCSailbot/sailbot_workspace/pull/170/files

**Comment by jahn18** [comment]
@patrick-5546 I updated the README. Should be ready to go once you are okay with it!

(also, forgive me for not fixing all the lint errors in the sailbot workspace dev container before pushing)

**Review by patrick-5546 — APPROVED** [review]
Looks great, thanks John!

I rebased this branch so that it only contains the commits relevant to this PR, I think some commits from the main branch got mixed in at some point. Now you can merge with main if you want, but in this case there's no need since this PR doesn't depend on any of the updates to main since the branch was created.

**Comment by jahn18** [comment]
@patrick-5546 Thanks Patrick!!

**Pull request merged**